.github/workflows/deploy.yml

```diff
@@ -0,0 +1,93 @@
+on:
+  workflow_call:
+    inputs:
+      image_version:
+        required: false
+        type: string
+        description: "Docker image version to pull for deployment"
+        default: "latest"
+      branch:
+        required: false
+        description: "Branch to pull in instance deployment"
```

Comment:
why do we need a branch in addition to the pre-built docker images ?

Reply:
cf. bellow